Archive/Imo/Imo2021Q1.lean

```diff
@@ -10,7 +10,7 @@ import Mathlib.Tactic.Linarith
 /-!
 # IMO 2021 Q1
 
-Let `n≥100` be an integer. Ivan writes the numbers `n, n+1,..., 2n` each on different cards.
+Let `n≥100` be an integer. Ivan writes the numbers `n, n+1, ..., 2*n` each on different cards.
```

Comment:
```suggestion
Let `n ≥ 100` be an integer. Ivan writes the numbers `n, n + 1, ..., 2 * n` each on different cards.
```

Reply:
I will change `n ≥ 100` but I prefer to keep the rest as it was.

Reply:
The general style consensus for Mathlib is to space out operators. I think our docstrings should reflect that too.

Reply:
I know. I prefer not to be the one who changes this tho.